### ISSUE TITLE
Research Summary: Towards A first step to understand flash loan and its application in Defi Ecosystem

### ISSUE THREAD

**Issue by twan-dev** [smartcontractresearchforum/research_content#253]
Author: Samuel @samuelngwu

Estimated date of completion: TBD

Link to Paper: https://yajin.org/papers/flashloan.pdf

Link to working draft: TBD

Source of approval: RP 15 05/28/21. Mentorship Program 2021.

**Comment by twan-dev** [comment]
- [x] SCRF has been notified to onboard the author
- [x] The ticket contains the following information
   - [x] Author
   - [x] An estimated date of completion
   - [x] Link to paper
   - [x] Link to draft
   - [x] Source of approval
- [x] Author does not already have another working document in Drafting

**Comment by samuelngwu** [comment]
@twan-dev  I worked on the comment made by Deane. I don't know if he has to get back to me before you can commence work otherwise you are free to start work on it.

**Comment by twan-dev** [comment]
- [ ] The summary is compliant with the running summary template
- [ ] A spelling and grammar check is complete
- [ ] If there exists an available Peer Reviewer, PM has notified them of this draft.

Currently waiting for the author to finish the first item on the list

**Comment by samuelngwu** [comment]
@twan-dev What "item on the list" am i suppose to finish

**Comment by twan-dev** [comment]
> @twan-dev What "item on the list" am i suppose to finish

As explained in my email to you, the summary isn't compliant with the running template. Feel free to reply to the email so that I can clarify if anything is unclear.

**Comment by samuelngwu** [comment]
This is your last email with me and from the content, you said the summary
is compliant with the running summary template.

Maybe you can guide me on the missing line.

Thank you
Samuel

**Comment by twan-dev** [comment]
@samuelngwu  Sure. I'll send you another email.

**Comment by twan-dev** [comment]
- [x] A first check on format and writing is complete.
- [x] A Peer Reviewer is ready to do reviews. (@mokawi)

**Comment by twan-dev** [comment]
- [x] All comments from Peer Reviewers have been discussed and addressed by authors. 
- [x] All suggestions provided by Peer Reviewers have been resolved.

**Comment by cipherix** [comment]
Good job on this @samuelngwu -- made some minor changes, moving it to editorial.

**Comment by samuelngwu** [comment]
Thank you for the corrections.


Samuel

**Comment by twan-dev** [comment]
@samuelngwu  @jamesmcgirk I've opened a HackMD file for collaboration: https://hackmd.io/YDq6eLBcQDKTmZ49ErsZDA

Please convert the summary draft into Markdown before starting editorial review, and make sure it's in good formatting before moving to the next stage. The HackMD should be the source of reference for what will be posted to the forum.

**Comment by samuelngwu** [comment]
Hello Twan,

Your email is acknowledged. I have not used this before and I am finding it
difficult to navigate through it.

Thank you
Samuel

**Comment by twan-dev** [comment]
> Hello Twan, Your email is acknowledged. I have not used this before and I am finding it difficult to navigate through it. Thank you Samuel


That's fine. We all did when we first started. But the forum renders posts with markdown language so it is necessary that you know how to use the tool before you post. 

You can start checking out the [research summary template](https://github.com/smartcontractresearchforum/docs/blob/main/en/content_research_summary_template.md) here, in "raw" mode, and then filling your summary into the template. You should also check out the guide provided by HackMD which comes with a new account. That should cover all of the essentials you would need to know!

**Comment by jamesmcgirk** [comment]
@samuelngwu there are a couple of good Google Doc to Markdown browser extensions that can make the process very simple. This is the one that I use: https://workspace.google.com/marketplace/app/docs_to_markdown/700168918607

**Comment by samuelngwu** [comment]
Thanks, I'll check it out.

**Comment by jamesmcgirk** [comment]
I made some edits in your word.doc, @samuelngwu

**Comment by samuelngwu** [comment]
Thank you. I have accepted it

**Comment by samuelngwu** [comment]
Hello Twan-dev,

This is my HackMD link for my research.
https://hackmd.io/YDq6eLBcQDKTmZ49ErsZDA?view

Thank you
Samuel

**Comment by jamesmcgirk** [comment]
Thanks so much for doing that for us, I’ll go in and check through it. Did I send you the go to market questions yet?

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by samuelngwu** [comment]
Hello James,

You are yet to send go to market questions.


Thank you

**Comment by jamesmcgirk** [comment]
added them into your original doc for you

**Comment by samuelngwu** [comment]
Should I include them in the HackMD doc?

**Comment by jamesmcgirk** [comment]
No, these are for the engagement team to create marketing materials with

**Comment by samuelngwu** [comment]
Thank you for the clarification.

I have responded to it.

Thank you

**Comment by jamesmcgirk** [comment]
@samuelngwu gtm complete, unless you have any comments or changes (check your discord)

**Comment by samuelngwu** [comment]
Thank you. I have checked it and there is no comment.

I have also checked my Discord and no message yet.


Thank you

**Comment by jamesmcgirk** [comment]
What’s your discord?

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by jamesmcgirk** [comment]
@twan-dev @zubepaul ready for your review and publication

**Comment by zubepaul** [comment]
@twan-dev @samuelngwu Engagement Review completed and approved for forum publication

**Comment by jamesmcgirk** [comment]
I’ll paste them in your original document once I return to my desk.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
